src/clj/services/question_service/p_question_service.clj

```diff
@@ -0,0 +1,23 @@
+(ns services.question-service.p-question-service)
+
+
+(defprotocol PQuestionService
+
+  (create-question!
+    [self question]
+    "Creates a course-iteration entry in the database and returns a course-iteration map.")
```

Comment:
I think it should be a "question entry".

Reply:
Yes, fixed the copy :spaghetti: mistake and replaced both occurrences of `course-iteration`  with `question`